### DISCUSSION TITLE
Starter feedback!

### DISCUSSION THREAD

**Discussion by RichieMcRichster** [eccentricdevotion/TARDIS#447] [Ideas]
Hello! Names Richard, just wanted to say before my feedback is I love ALL of the plugin, and I think its super cool! You've done an absolutely splendid job of making it, and I was just wanting to give a bit more feedback.

I think everything inside the plugin is very fun to use and play around with, but I do feel like a few small things could be a bit. . Tweaked. Really I'd just like a few more odd accessories, like the 2nd Doctors recorder, or the 11th's Fez. I think it'd also be really cool if there was a way to make custom mobs, like Daleks and Cybermen. This is also sort of just an option, but I think it'd be really cool if it'd be possible to make all the consoles a model instead of blocks, kind of like one of the options for the Chameleon circuit is a model of the Police Box, but with a wide set of colors to pick. I would like to mention that I've only been in this series for a couple weeks, and know nothing about coding. So I'm not sure if anything I am asking is a bit much, but I still think it'd just be cool to try :) then again some things for the plugin are not enabled on the server I use it on, so I may have to find a way to download the plugin itself for myself on a single world to look into every little thing you've made. With that being said though, thank you so much for making this plugin, I can tell you put in a lot of work into making it very accurate and fun to use, so thank you! :)

**Comment by eccentricdevotion** [comment]
Hi Richard,

Thanks for the feedback.

1. More accessories could be done, I generally like them to have some sort of purpose or function, rather than just cosmetic, so if you come up with any ideas, please make a separate issue.
2. Check out the TARDISWeepingAngels plugin for custom Who monsters - https://dev.bukkit.org/projects/tardisweepingangels/
3. Consoles as models could also be done, but you would lose a lot of the functionality as the models can't really be interacted with like the current setup - have you seen the new time rotors? https://eccentricdevotion.github.io/TARDIS/time-rotor.html

**Comment by RichieMcRichster** [comment]
Why thank you for responding, I was quite surprised to see this in my emails.

1. Firstly, I do remember that the Bow Tie and Headset don't actually work, as they don't look like what they look like. Both are just leather stained caps on my end (I have the resource pack and sound pack).
2. As for ideas, the 2nd Doctor's flute, right click to make an assortment of jukebox sounds would be really nice. I must admit that I'm still new to the series itself, but I think something silly like 7th's Umbrella, could open up and be used to glide down would be silly. Afterall, 7th's season was quite a comedic one (in my opinion).
3. I'll have to give that WeepingAngels pack a try as well, cause I feel like after a while there just isn't much to do with The Tardis, so maybe the extra looks and designs will give me more immersion.
4. As for the rotors, I have seen them indeed. As of now I am the 2nd Doctor, and set up my TARDIS to be like the 2nds, with the time rotor. I do think it's really cool, but it'd be **really** neat if you could make extra props. Say for example, the wooden chairs, or the golden pillars from the original 4's Tardis. Other things like the Astro - Map, which was shown
in 1sts on the Christmas special.
5. Another thing is I feel like the Screwdrivers should do a bit more? What they can do right now is neat, but maybe it's just because I always saw the 3rd use his as a weapon (or as close as you'd get to a Doctor using a weapon) so I always thought it should be able to fire a laser bolt, maybe not for damage, but to slow enemies down or just confuse them so you could run away.
6. One more thing, I'm not sure if any of the classic Doctors (before 4) or any of the other modern ones had a favorite food like 4 and 11, but if they do I'd really like to see them. I should also mention one of the "foods" is a paper bag, and it's just some battery icon, we couldn't figure out what it did so we left it be.

Thank you very much for responding to my suggestions, and I will surely look into the other pack, along with more parts of the plugin :)

**Comment by eccentricdevotion** [comment]
1. Bow ties and the communicator are working for me when I can equip them... Make sure you are using the latest https://github.com/eccentricdevotion/TARDIS-Resource-Pack - there does seem to be issues equipping them though which I'll look into when I have time
2. Maybe, they're something I'd use maybe once and then probably never again...
3. TARDISWeepingAngels is a plugin + a resource pack, you'll need both
4. Again a maybe, that's more of a resource pack thing than a plugin - other users of the TARDIS plugin have made their own resource packs to add extras
5. Not really into weapons that fire things - though you can have a Judoon guard if you have TARDISWeepingAngels...
6. ~~Paper bag model seems to be broken right now~~, I'll look into it. The paper bag is for storing jelly babies.

**Comment by eccentricdevotion** [comment]
Paper bag:
`SHIFT-click` the bag in inventory to store up to 4 stacks of Jelly Babies in it
`RIGHT-click` the bag in inventory to get a random Jelly Baby out of the bag

**Comment by eccentricdevotion** [comment]
Issues with bow-ties and communicators seem to be related to CREATIVE gamemode, try equipping them in SURVIVAL

**Comment by eccentricdevotion** [comment]
Paper bag model/textures re-added to TARDIS-Resource-Pack

**Comment by RichieMcRichster** [comment]
Why thank you very much. I'll be sure to update my pack, and keep looking
into the rest of the plugin. Thank you very much for your time :)
